_tutorials/page8.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: tutorial
+title: Nested Endpoints
+name: Nested Endpoints
+previous_page_title: Chaining Requests
+next_page_title: Default Values
+---
+
+# Nested Endpoints
+
+In order to clean and reuse our code, one `endpoint` can have multiple `endpoints`. This way,
```

Comment:
This phrase is confusing.

Maybe we could change it to something like: 

`ScanAPI supports a way to declare a base path, with multiple subsequent endpoints nested into it`. I don't know, just a suggestion. My suggestion is bad so, IDK. :rofl:

Reply:
I've updated it. Let's see if it is a bit more clear now :)